medusa/databases/main_db.py

```diff
@@ -559,3 +559,22 @@ def execute(self):
         if not self.hasColumn('tv_shows', 'plot'):
             self.addColumn('tv_shows', 'plot', 'TEXT', None)
         self.inc_minor_version()
+
+
+class AddResourceSize(AddPlot):
+    """Adds column resource_size to history table."""
+
+    def test(self):
+        """
+        Test if the version is at least 44.6
+        """
+        return self.connection.version >= (44, 6)
+
+    def execute(self):
+        backupDatabase(self.connection.version)
+
+        log.info(u"Adding column resource_size in history")
+        if not self.hasColumn("history", "resource_size"):
```

Comment:
imo anything only for a release should be `resource_*`, to be descriptive for anyone checking the db